pepper-apis/src/main/resources/changelog-master.xml

```diff
@@ -287,5 +287,6 @@
     <include file="db-changes/schema/DDP-7085-disabling-activity-block.xml" relativeToChangelogFile="true"/>
     <include file="db-changes/schema/DDP-7069-matrix-question-type.xml" relativeToChangelogFile="true"/>
     <include file="db-changes/schema/DDP-7292-activity-def-show-activity-status.xml" relativeToChangelogFile="true"/>
+    <include file="db-changes/schema/DDP-7306.xml" relativeToChangelogFile="true"/>
```

Comment:
This should be placed after the `DDP-7417-add-matrix-modal-columns` modification. Liquibase migrations are performed in the order they appear in the file, and this could introduce failures when performing a migration on a database which already has the matrix-modal column change added.

Reply:
Ok, I'll change the order, but as far as I know, Liquibase uses hashes that are stored in the DATABASESOMETHING table to avoid deduplication. So order really doesn't matter if the changes are not connected to each other (let's say change A adds column X and change B removes it).

Reply:
Done

Reply:
Thank you!